scripts/main.js

```diff
@@ -0,0 +1,83 @@
+const btnSubmit = document.querySelector('.submit');
+
+let bookObj;
+
+let bookid = 0;
+let author = ' ';
+let title = ' ';
+let bookListObj = [];
+let deserializedBookList;
+
+function saveData(dataObj) {
+  const dataString = JSON.stringify(dataObj);
+  localStorage.setItem('bookList', dataString);
+}
+
+function getData(key) {
+  return JSON.parse(localStorage.getItem(key));
+}
+
+function getBookList() {
+  let books = ' ';
+  deserializedBookList = getData('bookList');
+
+  deserializedBookList.forEach((value) => {
```

Comment:
@vechicin Can you please check again, because we are not facing an issue over here, you must have viewed an old pull request, we made a quick fix and do not have any issue?